Parameter 'value' may not be null message at Quarkus boot time.

Seeing this error due to a mis-configured property at boot time

`Caused by: java.lang.IllegalArgumentException: Parameter 'value' may not be null`

This has been discussed [here](https://github.com/quarkusio/quarkus/discussions/40215).

Make changes that provides useful context information to efficiently solve the issue.

--- a/core/deployment/src/main/java/io/quarkus/deployment/builditem/RunTimeConfigurationDefaultBuildItem.java
+++ b/core/deployment/src/main/java/io/quarkus/deployment/builditem/RunTimeConfigurationDefaultBuildItem.java
@@ -22,7 +22,7 @@ public final class RunTimeConfigurationDefaultBuildItem extends MultiBuildItem {
     public RunTimeConfigurationDefaultBuildItem(final String key, final String value) {
         Assert.checkNotNullParam("key", key);
         Assert.checkNotEmptyParam("key", key);
-        Assert.checkNotNullParam("value", value);
+        Assert.checkNotNullParam("value for key " + key, value);
         this.key = key;
         this.value = value;
     }

